test/src/main/java/org/apache/accumulo/test/GarbageCollectWALIT.java

```diff
@@ -72,7 +73,7 @@ public void test() throws Exception {
       try (Scanner scanner = c.createScanner(MetadataTable.NAME, Authorizations.EMPTY)) {
         scanner.forEach((k, v) -> {});
       }
-      UtilWaitThread.sleep(3 * 5_000);
+      Wait.waitFor(() -> countWALsInFS(cluster) == 2, SECONDS.toMillis(120), SECONDS.toMillis(15));
       assertEquals(2, countWALsInFS(cluster));
```

Comment:
Is this check redundant now?
```suggestion
```

Reply:
It is redundant, but I did't want to remove the only assert in that test.